### PR TITLE
Add Anmeldung bot, Hypofriend bot

### PR DESCRIPTION
**Pull request by nicbou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ A curated list of awesome Bots that help People in Berlin
 - [flatcrawler](https://github.com/grandchild/flatcrawler)
 - [flathunter](https://github.com/flathunters/flathunter)
 - [gis-berlin-rents](https://github.com/pawod/gis-berlin-rents)
+- [Hypofriend Telegram bot](https://hypofriend.de/en/lp/immofriend)
```
Comment (conradkirschner) [inline README.md]: Here I don't see any source code, therefore I would not put it in the list.

**Review by conradkirschner — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (conradkirschner) [inline README.md]:
```suggestion
```

**Review by conradkirschner — CHANGES_REQUESTED** [review]

**Comment by conradkirschner** [comment]
Thanks for contribution <3 ! 

After removing the one I would merge it :)

**Comment by nicbou** [comment]
Done!

**Review by conradkirschner — APPROVED** [review]

**Pull request merged**